### ISSUE TITLE
4_2 Sample Data Upgrade Test CX 2005

### ISSUE THREAD

**Issue by giancorderoortiz** [SAP/spartacus#14127]
Backend 2005 devci5

**Comment by giancorderoortiz** [comment]
pr: https://github.com/SAP/spartacus/pull/14128

**Comment by giancorderoortiz** [comment]
@KateChuen fyi

**Comment by giancorderoortiz** [comment]
new pr: https://github.com/SAP/spartacus/pull/14166

**Comment by giancorderoortiz** [comment]
Was blocked by: https://github.com/SAP/spartacus/issues/14149

**Comment by giancorderoortiz** [comment]
Retest taking place after cart validation removed as it's not compatible with cx 2005. 
See https://github.com/SAP/spartacus/pull/14160

**Comment by giancorderoortiz** [comment]
Task completed
